frontend/packages/core/src/Input/toggle-button-group.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import {
+  ToggleButtonGroup as MuiToggleButtonGroup,
+  ToggleButtonGroupProps as MuiToggleButtonGroupProps,
+} from "@material-ui/lab";
+
+export interface ToggleButtonGroupProps
+  extends Pick<
+    MuiToggleButtonGroupProps,
+    "value" | "children" | "size" | "orientation" | "onChange"
+  > {
+  /** If true, multiple children options can be selected simultaneously. */
+  multiple?: boolean;
+}
+
+const StyledMuiToggleButtonGroup = styled(MuiToggleButtonGroup)({
+  ".MuiToggleButton-root": {
+    "&.Mui-selected": {
+      color: "rgba(53, 72, 212, 1)",
```

Comment:
Oh I think I misread the designs when I made the original opacity comment. Not a huge deal either way but for posterity in case we want to update this later the hex value is `#3548D4`

Reply:
i thought it was the same color. ill change it

Reply:
It is, one is just a hex value vs rgba